### PR TITLE
fix: use dataset gens

### PR DESCRIPTION
**Pull request by cristianmtr**

### CODE REVIEW

File: stress-example/run_image.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+set -x
+
+if [ -z "$1" ]
+then
+  echo "No time (seconds) for running supplied. Use e.g. 'bash run_image.sh 60'" && exit 1
+fi
+
+bash ./env_vars_config.sh && \
```
Comment (JoanFM) [inline stress-example/run_image.sh:8]: let's have it one step at a time with some prints in between? Also we need to prepare different `query.yml` for each `type of indexer` (Faiss and Annoy)

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: stress-example/run_image.sh (hunk 1 above)
Comment (cristianmtr) [reply stress-example/run_image.sh:8]: What do you mean, "one step at a time"? Wasn't the intention of the script to run the indexing and querying in one command?

Comment (cristianmtr) [reply stress-example/run_image.sh:8]: Yes, I need to figure out how to best do that. Separate `query.yml`-s could be the way

File: stress-example/app.py [hunk 2]
```diff
@@ -9,8 +9,14 @@
 import click
 import requests
 
-# TODO make it available to the client
-FLOW_HOST_PORT = os.environ.get('FLOW_HOST_PORT')
+FLOW_PROTO = os.environ.get('FLOW_PROTO')
```
Comment (JoanFM) [inline stress-example/app.py:12]: can we have a default valu here?

File: stress-example/image/query_faiss.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+pods:
+  - name: encoder
+    uses: encoder.yml
+    shards: $JINA_SHARDS_ENCODER
+    host: $JINA_ENCODER_HOST
+    port_expose: 8000
+    timeout_ready: 600000
+  - name: vec_idx
+    uses: docker://jinahub/pod.indexer.annoyindexer:0.0.14-0.9.23
+    uses_internal: vec.yml
```
Comment (JoanFM) [inline stress-example/image/query_faiss.yml:17]: uses internal faiss_indexer.yml

File: stress-example/image/query_faiss.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  show_exc_info: true
+pods:
+  - name: encoder
+    uses: encoder.yml
+    shards: $JINA_SHARDS_ENCODER
+    host: $JINA_ENCODER_HOST
+    port_expose: 8000
+    timeout_ready: 600000
+  - name: vec_idx
+    uses: docker://jinahub/pod.indexer.annoyindexer:0.0.14-0.9.23
```
Comment (JoanFM) [inline stress-example/image/query_faiss.yml:16]: watch out here u are using annoy image

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: stress-example/app.py (hunk 2 above)
Comment (cristianmtr) [reply stress-example/app.py]: Done

File: stress-example/image/query_faiss.yml (hunk 3 above)
Comment (cristianmtr) [reply stress-example/image/query_faiss.yml]: Yes, fixed

File: stress-example/image/query_faiss.yml (hunk 4 above)
Comment (cristianmtr) [reply stress-example/image/query_faiss.yml]: Yes, thanks. Fixed

**Review by JoanFM — APPROVED** [review]

**Pull request merged**